sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/serializer/TypeRef.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.search.documents.implementation.serializer;
+
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+
+public abstract class TypeRef<T> {
```

Comment:
This can be removed as `TypeReference<T>` has been merged into Azure Core Experimental.